src/assets/css/nav.styl

```diff
@@ -0,0 +1,65 @@
+.navbar
+  margin-bottom: 0;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.3);
+  text-transform: uppercase;
+  background-color: #000000;
+
+  a
+    color: get-color('neutral','contrast');
+  li
+    & a:hover
+    & a:focus
+    &.active a
+      outline: none;
+      background-color: rgba(255, 255, 255, 0.2);
+    a
+      transition: background 0.3s ease-in-out;
+
+  .navbar-brand
+    width: 300px;
+    height: 50px;
```

Comment:
Looks like this is used in the content, do you think we should create a variable?

Reply:
will be fixed in a separate PR, https://github.com/sanisoclem/sanisoclem-dev/issues/9 has been raised for this.